assets/src/scss/new-identity/style.scss

```diff
@@ -10,3 +10,14 @@
   --block-columns--step-number--color: var(--grey-900);
   --block-columns--card-header--background: var(--p4-dark-green-800);
 }
+
+a.external-link, a.pdf-link {
+  color: var(--grey-900) !important;
+  text-decoration: none !important;
```

Comment:
You can override the current CSS variable like [here](https://github.com/greenpeace/planet4-master-theme/pull/1959/files#diff-b418b2241107b99a37ed0291b0385d4ad8fe3762dd4d859be83bf83c070428d7R20). But not necessary if we will merge the other PR.

Reply:
By doing this `--link--hover--text-decoration: none;`